webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,102 @@
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
-import Skeleton from 'react-loading-skeleton';
-import { ArrowIcon } from '@patternfly/react-icons';
-import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import React from 'react';
+import { SearchPlusIcon } from '@patternfly/react-icons';
 import {
+  Button,
   Card,
   DataList,
   DataListItem,
   DataListItemRow,
   DataListItemCells,
+  DataListWrapModifier,
   DataListCell,
+  CardActions,
   CardBody,
   CardTitle,
-  Accordion,
-  AccordionItem,
-  AccordionContent,
-  AccordionToggle,
+  CardHeader,
+  Popover,
 } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
 
 import { foremanUrl } from '../../../../foreman_tools';
 import { translate as __ } from '../../../common/I18n';
 import { useAPI } from '../../../common/hooks/API/APIHooks';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import AuditDetails from './AuditDetails';
 
 const AuditCard = ({ hostName }) => {
-  const [activeAccordion, setActiveAccordion] = useState(0);
-  const onToggle = id => {
-    if (id === activeAccordion) {
-      setActiveAccordion('');
-    } else {
-      setActiveAccordion(id);
-    }
-  };
-
-  const url = hostName && foremanUrl(`/api/audits?search=host+%3D+${hostName}`);
+  const url =
+    hostName && foremanUrl(`/audits?search=host+%3D+${hostName}&per_page=3`);
```

Comment:
Normally I'd say that you should have a function that accepts two parameters: hostName and per_page where the latter can be empty.

Then you can reuse it in the `<Link>` below. The `count` can be passed as a parameter (or defined as a constant). My programming professor in university would fail us if we used a number other than 0, 1 or 2 in code that wasn't a constant.

You should also URL encode hostName.

Reply:
`per_page` is constant here, the overview page in the host details page shows only recent resources.